Sources/SwiftDocC/Semantics/Reference/Row.swift

```diff
@@ -0,0 +1,98 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+public final class Row: Semantic, AutomaticDirectiveConvertible, MarkupContaining {
+    public let originalMarkup: BlockDirective
+    
+    @DirectiveArgumentWrapped(name: .custom("numberOfColumns"))
+    public private(set) var _numberOfColumns: Int? = nil
+    
+    @ChildDirective(requirements: .oneOrMore)
+    public private(set) var columns: [Column]
+    
+    static var keyPaths: [String : AnyKeyPath] = [
+        "_numberOfColumns"   : \Row.__numberOfColumns,
+        "columns"           : \Row._columns,
+    ]
+    
+    public var numberOfColumns: Int {
+        return _numberOfColumns ?? columns.map(\.size).reduce(0, +)
+    }
+    
+    override var children: [Semantic] {
+        return columns
+    }
+    
+    var childMarkup: [Markup] {
+        return columns.flatMap(\.childMarkup)
+    }
+    
+    @available(*, deprecated,
```

Comment:
Is it possible to use obsoleted here?

Reply:
Unfortunately not. 🫤

Obsoleted requires a version number: https://docs.swift.org/swift-book/ReferenceManual/Attributes.html.